public/fonts/fonts.css

```diff
@@ -1,4 +1,5 @@
 
+/* THIS FILE IS GENERATED BY fontscript.js */
```

Comment:
Should we even include this file in the git repo? I think not. Added this comment in the fontscript.js so it's clear this file shouldn't be edited manually